### PR TITLE
TypeScript Error

### PR DESCRIPTION
**Pull request by cleaner321**
Fix TypeScript error

src:
http://stackoverflow.com/questions/34120636/visual-studio-thinks-everything-is-a-typescript-resource

### CODE REVIEW

File: template/package.phone.appxmanifest [hunk 1]
```diff
@@ -40,11 +40,9 @@
                          Description="CordovaApp"
                          ForegroundText="light"
                          BackgroundColor="transparent">
-        <m3:DefaultTile Wide310x150Logo="images\Wide310x150Logo.png"
-                        Square71x71Logo="images\Square71x71Logo.png">
+        <m3:DefaultTile  Square71x71Logo="images\Square71x71Logo.png">
```
Comment (nikhilkh) [inline template/package.phone.appxmanifest]: Can you explain what this change is related to?

Comment (cadesalaberry) [inline template/package.phone.appxmanifest]: I am wondering the same...

But I would love the rest of the PR to be merged !

**Comment by cadesalaberry** [comment]
It seems like VisualStudio got their shit together and fixed it.

The following version does not need this PR:
```
Microsoft Visual Studio Community 2015
Version 14.0.25425.01 Update 3
Microsoft .NET Framework
Version 4.6.01038
```

**Comment by purplecabbage** [comment]
Can this be closed? Please confirm.

**Comment by cadesalaberry** [comment]
Yes.

**Pull request closed without merging**